src/intervals/arithmetic.jl

```diff
@@ -322,24 +322,28 @@ function mid{T}(a::Interval{T}, α)
 
     @assert 0 ≤ α ≤ 1
 
-    return (1-α) * a.lo + α * a.hi  # rounds to nearest
+    # return (1-α) * a.lo + α * a.hi  # rounds to nearest
+    return α*(a.hi - a.lo) + a.lo  # rounds to nearest
 end
 
-
-function mid{T}(a::Interval{T})  # specialized version for α=0.5
+function mid{T}(a::Interval{T})
 
     isempty(a) && return convert(T, NaN)
     isentire(a) && return zero(a.lo)
 
     a.lo == -∞ && return nextfloat(a.lo)
     a.hi == +∞ && return prevfloat(a.hi)
 
-    return 0.5 * (a.lo + a.hi)  # rounds to nearest
-end
+    # @assert 0 ≤ α ≤ 1
 
+    return simple_mid(a)
+end
 
 mid{T}(a::Interval{Rational{T}}) = (1//2) * (a.lo + a.hi)
 
+function simple_mid(a::Interval)
```

Comment:
Do we really need `simple_mid` as a new function? I think its sole use is inside that method for `mid`.

In any case, it is now good timing to add docstrings and tests.